roles/app_deployment/README.md

```diff
@@ -28,6 +28,39 @@ Role Variables
 | affinity  | UNDEF         | Describes node affinity scheduling rules for the vm. |
 | node_affinity | UNDEF     | Describes vm affinity scheduling rules e.g. co-locate this vm in the same node, zone, etc. as some other vms |
 | anti_affinity | UNDEF     | Describes vm anti-affinity scheduling rules e.g. avoid putting this vm in the same node, zone, etc. as some other vms. |
+| image | UNDEF | Dictionary specifing the image of the virtual machine. The dictionary can contain <i>url</i> and <i>storage</i> keys. <i>url</i> defines for qcow image to be used as datavolume source and <i>storage</i> defines the size of the volume to be created. |
+| wait | true | If <i>true</i> wait for the virtual machines to be in requested state. |
```

Comment:
I see that everywhere we use single form so let's be consistent and change to `the virtual machine to be in`